app/javascript/src/pages/conversations/Sidebar.tsx

```diff
@@ -19,14 +19,43 @@ function localeDate(date) {
   return new Date(date).toLocaleString();
 }
 
+export const eventsSubscriber = (appId, cableApp, callback) => {
```

Comment:
Hi @johnny-hsiao I still not sure why is there is a new subscription to cable instead of leveraging the events provided on AppContainer.tsx.